hi-ml-azure/src/health_azure/himl.py

```diff
@@ -381,11 +483,59 @@ def _str_to_path(s: Optional[PathOrString]) -> Optional[Path]:
     return s
 
 
+def create_v2_inputs(ml_client: MLClient, input_datasets: List[DatasetConfig]) -> Dict[str, Input]:
+    """
+    Create a dictionary of Azure ML v2 Input objects, required for passing input data in to an AML job
+
+    :param ml_client: An MLClient object.
+    :param input_datasets: A list of DatasetConfigs to convert to Inputs.
+    :return: A dictionary in the format "input_name": Input.
+    """
+    inputs: Dict[str, Input] = {}
+    for i, input_dataset in enumerate(input_datasets):
+        input_name = f"INPUT_{i}"
+        version = input_dataset.version or 1
+        data_asset: Data = ml_client.data.get(input_dataset.name, version=str(version))
+        data_path = data_asset.id or ""
+        # Note that there are alternative formats that the input path can take, such as:
+        # v1_datastore_path = f"azureml://datastores/{input_dataset.datastore}/paths/<path_to_dataset>"
+        # v2_dataset_path = f"azureml:{input_dataset.name}:1"
+
+        inputs[input_name] = Input(  # type: ignore
+            type=AssetTypes.URI_FOLDER,
+            path=data_path,
+            mode=InputOutputModes.MOUNT,
+        )
+    return inputs
+
+
+def create_v2_outputs(output_datasets: List[DatasetConfig]) -> Dict[str, Output]:
+    """
+    Create a dictionary of Azure ML v2 Output objects, required for passing output data in to an AML job
```

Comment:
```suggestion
    Create a dictionary of Azure ML v2 Output objects, required for passing output data into an AML job
```

Reply:
Here we are passing the data to the job, so pass 'in to' is correct (https://idioms.thefreedictionary.com/pass+into)